manifests/params.pp

```diff
@@ -35,7 +36,11 @@
         default            => 'bsd-mailx',
       }
 
-      $master_os_template = "${module_name}/master.cf.debian.erb"
+      if $::lsbdistcodename == 'xenial' and $chroot_daemons {
```

Comment:
The Boolean is false unless both conditions are met, which means no change in behaviour from present if the variable is not set (ie - the default master.cf.debian.erb is shipped).

Future debian & RHEL releases will have postfix3, so you may need to update this logic when they are released.